ParkPlanner/Pages/Index.cshtml

```diff
@@ -3,7 +3,7 @@
 @using ParkPlannerPark
 @{
     ViewData["Title"] = "Home page";
-    List<Datum> park = (List<Datum>)ViewData["Park"];
+    List<Datum> parks = (List<Datum>)ViewData["Park"];
```

Comment:
This is datum about an individual park, so I could go either way on this variable name.